### ISSUE TITLE
Wrong test in project euler 462

### ISSUE THREAD

**Issue by ilenia-magoni** [freeCodeCamp/freeCodeCamp#57724]
https://github.com/freeCodeCamp/freeCodeCamp/blob/1df0e61091596a5d16377d15a252c1d34ccf653c/curriculum/challenges/english/18-project-euler/project-euler-problems-401-to-480/problem-462-permutation-of-3-smooth-numbers.md#L32-L33

There should be a comma separating the two terms, instead of having the `===` operator

**Comment by a2937** [comment]
I have a question. Now normally when a typo like that is made, I would expect it to error out. But I only see five past and one as a pending when I test the file. When I test the specific block, I get 79 pending challenges. 

I presume that there might be a good bit of challenges in that Superblock that might need looked at it, that is if you can replicate this behavior. My question is if you can replicate that behavior and are getting similar results.

**Comment by a2937** [comment]
I just now looked at it. This is a problem though because with no solution there's no automatic indicators that our tests are broken like this.

**Comment by a2937** [comment]
![Expected Output](https://github.com/user-attachments/assets/a781584a-f7b8-4463-aa6f-f65eaa9949d8)

This is what the output would look like if there's a solution intended to solve the problem.

**Comment by ilenia-magoni** [comment]
Yes, most of project euler and rosetta code doesn't have solutions. They would need to be written in a way to not timeout, these problems get easily resource intensive, and they would increase a lot the time for testing all the curriculum

**Comment by a2937** [comment]
You're right. Four hundred plus challenges that are resource intensive would increase the amount of time the curriculum checker to execute. 

I'd love to try to take on the task for writing efficient solutions for the Project Euler and Rosetta code challenges though. Maybe even learn something along the way.

**Comment by ilenia-magoni** [comment]
@a2937 I don't think it is necessary, it is not a part of the curriculum that is much used, see needing 3 years to figure out the bug
